doc/source/reference/random/parallel.rst

```diff
@@ -156,6 +157,8 @@ are listed below.
 +-----------------+-------------------------+-------------------------+-------------------------+
 | PCG64           | :math:`2^{128}`         | :math:`~2^{127}` ([3]_) | 64                      |
 +-----------------+-------------------------+-------------------------+-------------------------+
+| PCG64DXSM       | :math:`2^{128}`         | :math:`~2^{127}` ([3]_) | 64                      |
++-----------------+-------------------------+-------------------------+-------------------------+
 | Philox          | :math:`2^{256}`         | :math:`2^{128}`         | 64                      |
 +-----------------+-------------------------+-------------------------+-------------------------+
 
```

Comment:
I find two things unclear here.

- AFAICT, the single draw is the corresponding "Bits". Maybe relabel "Bits in Draw'.
- How is the jump size of the PCG family almost the cycle size without causing problems.

Reply:
The point is to not have overlapping sequences of draws (and avoid the bad spots induced by the LCG, like a `2**N` would do), not necessarily to avoid jumping over your nominal "0" point on the cycle as you jump. As long as you are jumping _over_ and not _into_, you're fine. Using the Golden Ratio for that purpose is a [well-attested approach](http://extremelearning.com.au/unreasonable-effectiveness-of-quasirandom-sequences/) for spreading yourself out over cyclical structures.

Reply:
Thanks for the link.